index.html

```diff
@@ -242,7 +242,6 @@ <h1 class="Header">About Me</h1>
         <hr class="AboutLine2">
         <div class="MySkills">
           <ul>
-
```

Comment:
Great job on your form! It's almost perfect and meets all the requirements. I just have a **few friendly suggestions to make your portfolio even better:**

- [ ] **[OPTIONAL]** You might want to consider using a different heading tag instead of h1 for the text in your contact form. **This can improve SEO and using multiple h1 tags isn't generally recommended.** 😉

- [ ] Could you please **fix the alignment of the text in your button?** It's not quite centered at the moment. 🙋‍♂️

- [ ] Additionally, it would be great if **you could add disabled, active, and hover effects to your buttons** for a more polished look. 🔍

   ![image](https://github.com/bhushan354/Portfolio_Project/assets/107952724/d0858163-cb71-489c-97ed-19e1ee77f08c)